README.md

```diff
@@ -3,16 +3,51 @@
 Getline Network
 ===============
 
-This is the main repository for the Getline Distributed P2P Lending Platform.
+This is the main repository for the Getline Distributed P2P Lending Platform. 
+
+Web interface
+-------------
+
+For a work-in-progress webapp compatible with Metamask, see [https://demo.getline.in](demo.getline.in).
+
+This repository
+---------------
 
 We are currently under heavy development.
 
 Structure is as follows:
   - `dapp/` - Solidity smart contracts for loans.
-  - `frontend/` - Frontend library and client.
+  - `frontend/` - Frontend webapp.
   - `metabackend/` - Loan metadata and bytecode cache server.
+  - `getline.ts/` - Official client library in Typescript.
   - `pb/` - Protobuf definitions for the metabackend API.
 
 We also have some supporting directories:
   - `vendor/` - vendorified dependencies, currently for the metabackend only
   - `production/` - Dockerfiles and Kubernetes description files for hosting the metabackend
+
+
+API
+---
+
+We highly recommend using `getline.ts` for interacting with Getline from external code.
+
+Otherwise, you are free to call smart contract methods and the metabackend via gRPCWeb.
+
+Versioning
+----------
+
+**As Getline is currently in proof-of-concept phase on Rinkeby, we do not plan on keeping any stable API before a mainnet release.**
+
+Once we reach version 1.0.0, we will provide API versioning at the following levels:
+
+ - `getline.ts` - Forwards and backwards compatible API for all externel objects, semver major bump otherwise.
+ - Smart contracts - Versioned by ABI, with all versions of smart contracts sources, bytecode and ABI definitions provided by the metabackend.
+ - Protobufs - Forwards and backwards compatible wire format for all messages and RPC calls, semver major bump otherwise.
+
+Development
+-----------
+
+All development and issue tracking happens on this repository. Pull requests are welcome, but keep in mind that this is a very fast moving codebase.
```

Comment:
Pierwsze pytanie jakie padnie w głowie programisty: gdzie mam zajrzeć/co zrobić aby postawić to u siebie